$NetBSD: patch-src_dune,v 1.2 2024/02/14 15:59:56 wiz Exp $

Avoid double rule for cairo_ocaml.h
(other "rule" introduced by creating cairo_ocaml.h manually
in pkgsrc Makefile).

--- src/dune.orig	2024-02-14 15:40:40.921184880 +0000
+++ src/dune
@@ -10,6 +10,6 @@
  (synopsis "Binding to Cairo, a 2D Vector Graphics Library"))
 
 (rule
- (targets c_flags.sexp c_library_flags.sexp cairo_ocaml.h)
+ (targets c_flags.sexp c_library_flags.sexp)
  (deps    cairo_ocaml.h.p)
  (action  (run ../config/discover.exe)))
